### PR TITLE
#76 issue fixed

### PR DESCRIPTION
**Pull request by d-molotchko**
Please review this. I believe there is qulice error in src/main/java/org/takes/facets/flash/RsFlash.java in lines 95, 96. Should I create issue?

### CODE REVIEW

**Comment by davvd** [comment]
Thanks for your pull request, let me find someone who can review it

**Comment by davvd** [comment]
@longtimeago review this one, please

**Comment by longtimeago** [comment]
@d-molotchko checkstyle is not happy :)

**Comment by longtimeago** [comment]
@d-molotchko Also I don't see the test. Did you reproduce the issue first?

**Comment by d-molotchko** [comment]
@longtimeago I fix #76 issue but get qulice check error on my computer. AppVeyor does not have this check error. It has error described in #79 
What's wrong with checkstyle?

**Comment by longtimeago** [comment]
```
[ERROR] /src/main/java/org/takes/facets/flash/RsFlash.java[95]: method call rparen at indentation level 16 not at correct indentation, 12 (IndentationCheck)
[ERROR] /src/main/java/org/takes/facets/flash/RsFlash.java[96]: operator new rparen at indentation level 12 not at correct indentation, 8 (IndentationCheck)
```

File: pom.xml [hunk 1]
```diff
@@ -152,7 +152,6 @@
         <dependency>
             <groupId>com.google.guava</groupId>
             <artifactId>guava</artifactId>
-            <scope>test</scope>
```
Comment (yegor256) [inline pom.xml]: we can't do this. the library promises that there are no third-party dependencies. we can't use Guava in production code, only in tests

**Comment by yegor256** [comment]
@d-molotchko one comment above from me, Guava can't be used in main classes

**Comment by longtimeago** [comment]
@yegor256 looks like now PR is clean

**Comment by yegor256** [comment]
@rultor merge

**Comment by longtimeago** [comment]
@d-molotchko please, merge your branch from master

**Comment by d-molotchko** [comment]
@longtimeago merged

**Comment by longtimeago** [comment]
@rultor merge please

**Comment by yegor256** [comment]
@d-molotchko I think that we can use only `new SimpleDateFormat(` for string formatting. `StringBuilder` is redundant. Something like this: `new SimpleDateFormat("'Expires='EEE, d MMM yyyy HH:mm:ss Z", ...)`. What do you think?

**Comment by d-molotchko** [comment]
@yegor256 Yes, it's the simpliest way. Done.

**Comment by yegor256** [comment]
@d-molotchko why do you use different formatting/indentation in these two almost identical pieces of code?

**Comment by d-molotchko** [comment]
@yegor256 I can't use normal indentation in RsFlash.java because I got qulice check error on it. I created issue in qulice project.

**Comment by yegor256** [comment]
@rultor try to merge this

**Comment by d-molotchko** [comment]
@yegor256 merged from master

**Comment by yegor256** [comment]
@d-molotchko check this pull request: https://github.com/yegor256/takes/pull/78/files You see how `String.format` is used there together with `Locale.ENGLISH`? I think this method looks better than the one you introduced. Don't you think?

**Comment by d-molotchko** [comment]
@yegor256 you are right. But we must replace plain GMT substring with timezone shift. Right?

**Comment by yegor256** [comment]
@d-molotchko I'm not sure, not an expert there... we should check with RFC

**Comment by d-molotchko** [comment]
@yegor256 I think that this time string represents server time. But server may locates in different timezones. I think there may be cases where a client get expired cookies time.

**Comment by d-molotchko** [comment]
@yegor256 [rfc7231](https://tools.ietf.org/html/rfc7231#section-7.1.1.1) defines time format as a fixed-length and single-zone subset of the date and time specification used by the Internet Message Format. So GMT substring is valid, but keep in mind that local server time must be converted to GMT timezone before making HTTP headers.

**Comment by yegor256** [comment]
@rultor merge

**Comment by yegor256** [comment]
@rultor merge again

**Comment by elenavolokhova** [comment]
@d-molotchko According to our [quality rules](http://www.teamed.io/qa.html): 
> Pull request title explains the problem it is fixing.

This title doesn't describe an issue. Please, try to find better title.

> Pull request description explains the solution proposed and contains a link to the original ticket it is related to.

Description provided points to another issue. Pull request must briefly explain what was done in the branch, so that we can use it in future. Please, add such information along with the link to original ticket.

**Comment by elenavolokhova** [comment]
@d-molotchko Thank you!

**Comment by elenavolokhova** [comment]
@davvd Ticket passed QA review. Quality is acceptable.

**Comment by davvd** [comment]
> @davvd Ticket passed QA review. Quality is acceptable.

@elenavolokhova thank you

**Comment by davvd** [comment]
@longtimeago 10 mins was added to the account of @elenavolokhova (for QA review), in transaction 54988145; **21 mins** sent to your balance (ID `54988168`), many thanks! It took 119 hours and 47 mins.; you're getting extra minutes here (c=6); added +21 to your rating, now it is equal to [+3751](http://www.netbout.com/b/35024?open=rating)

**Pull request merged**